modules/salt/pages/formula-imagesync.adoc

```diff
@@ -17,3 +17,11 @@ This ensures that you do not have to perform image synchronization manually, how
 Alternatively, you can leave this list blank to have all images synchronized.
 . Click btn:[Save Formula] to save your configuration.
 . Apply the highstate.
+
+
+[NOTE]
+====
+The Image Synchronization state does not delete cached downloaded images.
```

Comment:
"cached downloaded" sounds strange.  If it's a cache you probably can remove "downloaded".